extensions/kdn/scripts/build.cjs

```
@@ -37,6 +37,11 @@ if (fs.existsSync(builtinDirectory)) {
 
 const zip = new AdmZip();
 zip.addLocalFolder(path.resolve(__dirname, '../'));
+zip.deleteFile('assets/');
+zip
+  .getEntries()
+  .filter(e => e.entryName.startsWith('assets/'))
+  .forEach(e => zip.deleteFile(e.entryName));
```

Biome (2.4.15)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)